perf/smallbank_workload/src/main.rs

```diff
@@ -16,6 +16,7 @@
  */
 extern crate clap;
 extern crate crypto;
+extern crate env_logger;
```

Comment:
We used `simple_logging` in the Seth RPC code: See https://github.com/hyperledger/sawtooth-seth/blob/master/families/seth/rpc/src/main.rs